runatlantis.io/docs/using-atlantis.md

```diff
@@ -104,3 +104,31 @@ Because Atlantis under the hood is running `terraform apply plan.tfplan`, any Te
 They're ignored because they can't be specified for an already generated planfile.
 If you would like to specify these flags, do it while running `atlantis plan`.
 
+---
+## atlantis import
+
+```bash
+atlantis import [options] -- [terraform import flags] addr id
+```
+### Explanation
+Runs `terraform import` for the plan that matches the directory/project/workspace.
+
+### Examples
+```bash
+# Runs import for single plan to import a resource.
+atlantis import -- addr id
```

Comment:
As of PR description, this implementation doesn't handle `atlantis import addr id`. This is because command_parser is not easy to handle these params

Reply:
I change the parser impl. In this impl, I append args into `extraArgs`. please check again. @Fabianoshz @runatlantis/maintainers

https://github.com/runatlantis/atlantis/pull/2783/commits/39158d4810a7e9bba722f2d87eb78f9bcefea4d6

Reply:
cc @nitrocode @jamengual

Reply:
Yes. This is much more intuitive to the user. Glad you got this to work without using the `--`